src/test/java/task3/Calculator.java

```diff
@@ -0,0 +1,79 @@
+package task3;
+
+import java.lang.*;
+import java.util.Scanner;
+
+public class Calculator {
+    private int solution;
+    private static int x;
+    private static int y;
+    private char operators;
+
+
+    public Calculator()
+    {
+        solution = 0;
+        Scanner operators = new Scanner(System.in);
+        Scanner operands = new Scanner(System.in);
+    }
+
+
+
+    public int addition(int x, int y)
+    {
+        return x + y;
+    }
+    public int subtraction(int x, int y)
+    {
+        return x - y;
+    }
+    public int multiplication(int x, int y)
+    {
+        return x * y;
+    }
+    public int division(int x, int y)
+    {
+        solution = x / y;
+        return solution;
```

Comment:
why do you use 'solution' only here? are you sure that you need it? In case if YES, add it same for above methods